dev/package.sh

```diff
@@ -45,5 +45,10 @@ elif [ "$LINUX_OS" == "centos" ]; then
     process_setup_centos_7
   fi
 fi
+elif [ "$LINUX_OS" == "alinux" ]; then
```

Comment:
the format seems wrong here:

```
./dev/package.sh: line 47: syntax error near unexpected token `elif'
./dev/package.sh: line 47: `elif [ "$LINUX_OS" == "alinux" ]; then'
Build step 'Execute shell' marked build as failure
```

Reply:
Fire a Pr to fix it. https://github.com/oap-project/gluten/pull/1342